python/qpid_dispatch_internal/tools/display.py

```diff
@@ -138,6 +143,33 @@ def num(self, value):
     value /= 1000
     return self.numCell(value, 'g')
 
+def PlainNum(value):
```

Comment:
Is the precision check even necessary since the return string will be the minimum size in any case?  Could PlainNum simply return "%d" % value ?

Reply:
Yes, it could simply return   "%d" % value. I added the precision because I thought the numbers were not left justified if  "%d" % value was used but I was wrong. I ran the tests with "%d" % value. and the values came out correctly.